hasjob/models/jobpost.py

```diff
@@ -587,6 +588,17 @@ def viewstats_helper(jobpost_id, interval, limit, daybatch=False):
     cviewed = batches * [0]
     copened = batches * [0]
     capplied = batches * [0]
+    cbuckets = batches * ['']
+
+    now_local = datetime.now()
+    for delta in range(batches):
+        if daybatch:
+            # here delta=0, so last item is the latest date/hour
+            cbuckets[batches - delta - 1] = (now_local - timedelta(days=delta)).strftime("%b %d")
+        else:
+            from_hour = (now_local - timedelta(hours=delta)).strftime("%H:00")
+            to_hour = (now_local - timedelta(hours=delta - 1)).strftime("%H:00")
```

Comment:
Looks like Babel does have a `format_datetime` method that we should be using instead of strftime. The locale is already set for you by Baseframe. https://pythonhosted.org/Flask-Babel/#formatting-dates